coins/monero/src/wallet/mod.rs

```diff
@@ -121,12 +123,19 @@ impl ViewPair {
       .concat(),
     ))
   }
+
+  /// returns the MoneroAddress type according to specifications in the meta
+  pub fn address(&self, meta: AddressMeta<MoneroAddressBytes>) -> MoneroAddress {
+    MoneroAddress::new(meta, self.spend, self.view.deref() * &ED25519_BASEPOINT_TABLE)
+  }
 }
 
 /// Transaction scanner.
-/// This scanner is capable of generating subaddresses, additionally scanning for them once they've
-/// been explicitly generated. If the burning bug is attempted, any secondary outputs will be
-/// ignored.
+/// - This scanner is capable of generating subaddresses, additionally scanning for
+/// them once they've been explicitly generated by using `subaddress()` member function.
+/// It wont be able to track the outputs for subaddresses that aren't generated this way.
+/// - If the burning bug is attempted, any secondary outputs will be ignored.
+/// or standard address type if none passed.
```

Comment:
This line isn't valid. The prior statement was if the burning bug is attempted, the secondary outputs will be ignored. That requires using a standard, burning bug vulnerable address. If None is passed for the burning_bug, immune addresses will be used and the burning bug can't be attempted. If Some is passed, it can be attempted, yet the secondaries will be ignored. That's just not the "or" conditions.